docs/api-markdown-documenter/build-api-nav.js

```diff
@@ -63,11 +67,11 @@ async function buildNavBar(documents, version) {
 }
 
 const saveToFile = async (filename, version, data) => {
-	if (!fs.existsSync(path.join(__dirname, "..", "data", version))) {
-		fs.mkdirSync(path.join(__dirname, "..", "data", version), { recursive: true });
+	if (!fs.existsSync(path.join(dirname, "..", "data", version))) {
+		fs.mkdirSync(path.join(dirname, "..", "data", version), { recursive: true });
 	}
 	fs.writeFile(
-		path.join(__dirname, "..", "data", `${version}/${filename}`),
+		path.join(dirname, "..", "data", `${version}/${filename}`),
```

Comment:
nit

```suggestion
		path.join(dirname, `../data/${version}/${filename}`),
```

Reply:
either pass them all as individual args, or join them all in the string template.

Reply:
Good call. But we can even go one level further.
```suggestion
		path.join(dirname, "..", "data", version, filename),
```